packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -18,6 +18,7 @@ type ReadyProps = {
 };
 
 export type ViewNameType =
+  | 'complete-signin'
```

Comment:
Does the CompleteSignin page use the Ready component? I don't see it referenced elsewhere.

Reply:
Oh thanks for catching that, I meant to remove that!